### DISCUSSION TITLE
Omega Grain rotation angle from aspect ratio [omega,a,b] = grains.fitEllipse

### DISCUSSION THREAD

**Discussion by snayir6** [mtex-toolbox/mtex#1024] [Ask Anything] [answered]
Hello Ralf and Rudiger,

I am using the fitEllipse function to extract the individual long and short diameter/axes (a,b) measurement from each grain. Omega is described as the rotation of the ellipse but according to which origin? Would you give me more description
Thanks,
Selda

`[omega,a,b] = grains.fitEllipse`

**Comment by kilir** [comment, marked as answer]
Hi,
the angle omega is with respect to x in the right handed x,y,z coordinate system used in Mtex e.g to define points on an EBSD map or vector3 for example.
Hope that helps.
